static/app/views/dashboards/widgets/timeSeriesWidget/markDelayedData.tsx

```diff
@@ -1,24 +1,24 @@
-import type {TimeseriesData} from '../common/types';
+import type {TimeSeries} from '../common/types';
 
 /**
  * Given a timeseries and a delay in seconds, goes through the timeseries data, and marks each point as either delayed (data bucket ended before the delay threshold) or not
  */
 
-export function markDelayedData(timeserie: TimeseriesData, delay: number) {
+export function markDelayedData(timeSeries: TimeSeries, delay: number) {
```

Comment:
```suggestion
export function markDelayedTimeSeries(timeSeries: TimeSeries, delay: number) {
```
Seems like you also have a couple different functions here, but the files is called markDelayedData, is there a better name for this maybe?

Reply:
I want to punt this until a little later (there are other helpers like `shiftTimeserieToNow`) to keep this PR smaller